src/effect/fadeInDown.js

```diff
@@ -14,16 +14,14 @@ export default {
     'panel': {
         'position': 'fixed',
         'backgroundColor': '#fff',
-        'width': '400px',
-        'height': '300px',
+        'width': 'auto',
+        'height': 'auto',
         'top': '50%',
         'left': '50%',
-        'marginTop': '-150px',
-        'marginLeft': '-200px',
         'borderRadius': '5px',
         'boxSizing': 'border-box',
         'boxShadow': '0 2px 4px rgba(0, 0, 0, 0.3)',
-        'transform': 'translate3d(0, 0, 0)',
+        'transform': 'translate3d(-50%, -50%, 0)',
```

Comment:
Sorry, to not have found time earlier to respond. 

I have found a possible solution to the transform problem that I would check in if you approve it here. 

The transform you figured out to be problematic is `'transform': translate(0, -100px, 0)`.

I would change that line to `'transform': translateY(-100px) translate(-50%, -50%, 0)`.

In modern browsers this could also be shortened to `'transform': translate(0, calc(-100px - 50%,) 0)` which unfortunately does not work in iOS webviews so I would go with the first proposal.

Of course, for the other transitions this would have to be adapted accordingly.